panel.js

```diff
@@ -333,7 +295,8 @@ function createCardsBrandSection(brand, cards) {
   var table = $('<table>');
   $.each(cards, function (index, item) {
 
-    if (!item.favourite) {
```

Comment:
Q: Can `item` be `null`/`undefined` here?

Reply:
It cannot be in this case. Anyhow, with the `try-catch` added above the loading of the extension will not fail in case of invalid data or an unexpected error.